modules/channel-web/assets/inject.js

```diff
@@ -23,12 +23,12 @@ window.addEventListener('message', function(payload) {
 })
 
 function init(config) {
-  const host = config.host || ''
+  const host = config.host || window.ROOT_PATH || ''
   const botId = config.botId || ''
-  const cssHref = host + '/assets/modules/channel-web/inject.css'
+  const cssHref = host + 'assets/modules/channel-web/inject.css'
   injectDOMElement('link', 'head', { rel: 'stylesheet', href: cssHref })
   const options = encodeURIComponent(JSON.stringify({ config: config }))
-  let iframeSrc = host + '/lite/' + botId + '/?m=channel-web&v=Embedded&options=' + options
+  let iframeSrc = host + 'lite/' + botId + '/?m=channel-web&v=Embedded&options=' + options
```

Comment:
we could let the `/lite/` live and replace potential double slashes, i.e. 

```js
const SanitizeUrl = url => 
  url.replace(/^(http|https):\/\/(.+)/i, (_full, protocol, rest) => protocol + "://" + rest.replace(/\/\//ig, '/'))
```

Reply:
Shouldn't sanitization be done only once when it is set , instead of everywhere where we load an URL ? Seems a bit like overkill to me